webapps/assembly/src/main/java/org/camunda/bpm/webapp/impl/security/filter/CsrfPreventionFilter.java

```diff
@@ -46,20 +46,23 @@
  * Provides basic CSRF protection implementing a Same Origin Standard Header verification (step 1)
  * and a Synchronization Token with a cookie-stored token on the front-end.
  *
+ * This token is called XSRF-TOKEN, generated by the server, sent in the first response as a Cookie,
+ * then stored as a client Cookie, and sent back as an HTTP header (X-XSRF-TOKEN) on subsequent requests
```

Comment:
To make it appear in a new line:

```suggestion
 * <p>
 * This token is called XSRF-TOKEN, generated by the server, sent in the first response as a Cookie,
 * then stored as a client Cookie, and sent back as an HTTP header (X-XSRF-TOKEN) on subsequent requests.
```